sdk/communication/Azure.Communication.Common/CHANGELOG.md

```diff
@@ -1,10 +1,10 @@
 # Release History
 
-## 2.0.0-beta.2 (Unreleased)
+## 2.0.0 (Unreleased)
 
 ### Features Added
-
-### Breaking Changes
+- Added support for a new communication identifier `MicrosoftTeamsAppIdentifier`.
+- Introduction of `MicrosoftTeamsAppIdentifier` is a breaking change. It will affect code that relied on using `UnknownIdentifier` with a rawID starting with `28:orgid:`.
```

Comment:
```suggestion

### Breaking Changes
- Introduction of `MicrosoftTeamsAppIdentifier` is a breaking change. It will affect code that relied on using `UnknownIdentifier` with a rawID starting with `28:orgid:`.
```

Reply:
let's mention other gov.environments prefixes

Reply:
Mentioned.